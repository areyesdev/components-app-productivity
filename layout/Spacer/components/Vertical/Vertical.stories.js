import { Vertical } from '.'
import { options, styles } from '../..'

import {
  getTemplate,
  getListTemplate,
  getOptionsArgTypes,
} from '../../../../helpers/storybook'

const Template = getTemplate(Vertical, styles)
const ListTemplate = getListTemplate(Vertical, styles)

export default {
  title: 'Layout/Spacer/Vertical',
  component: Vertical,
  args: { isPlayground: true },
  argTypes: {
    size: getOptionsArgTypes(options.sizes),
  },
}

export const Default = Template.bind({})

export const Sizes = ListTemplate.bind({})
Sizes.args = { items: options.sizes.map((size) => ({ size })) }
