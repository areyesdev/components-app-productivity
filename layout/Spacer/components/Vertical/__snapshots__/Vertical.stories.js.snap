// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Layout/Spacer/Vertical Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="spacer vertical is-playground"
    style={
      Object {
        "height": 0,
      }
    }
  />
</div>
`;

exports[`Storyshots Layout/Spacer/Vertical Sizes 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="spacer vertical is-playground"
    style={
      Object {
        "height": 0,
      }
    }
  />
  <div
    className="spacer vertical is-playground"
    style={
      Object {
        "height": 5,
      }
    }
  />
  <div
    className="spacer vertical is-playground"
    style={
      Object {
        "height": 15,
      }
    }
  />
  <div
    className="spacer vertical is-playground"
    style={
      Object {
        "height": 20,
      }
    }
  />
  <div
    className="spacer vertical is-playground"
    style={
      Object {
        "height": 40,
      }
    }
  />
</div>
`;
